pygmt/tests/test_grdimage.py

```diff
@@ -117,6 +117,22 @@ def test_grdimage_shading_xarray(grid, shading):
     return fig_ref, fig_test
 
 
+@check_figures_equal()
```

Comment:
What do you think about adding an xfail here (so we can merge this PR into master and let Paul test things easier in https://github.com/GenericMappingTools/gmt/issues/5294)?
```suggestion
@pytest.mark.xfail(
    reason="Incorrect scaling of geo CPT on xarray.DataArray grdimage plot."
    "See https://github.com/GenericMappingTools/gmt/issues/5294",
)
@check_figures_equal()
```